content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,97 @@
+
+There are two major features which are marked as experimental stage but might be useful in some specific scenarios.
+ 
+- Support [signing, listing and verification with OCI image layout](https://notaryproject.dev/docs/how-to/oci-image-layout/) for signing images before they are pushed to a registry. It enables users to sign and verify images in an air-gapped environment.
+- Support [OCI distribution referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc2/spec.md#enabling-the-referrers-api), which allows Notation CLI to fetch a list of referrers in an efficient and clean manner.
+
+### Better extensibility and more plugins support
+
+Notation has an [extensible plugin framework design](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/plugin-extensibility.md) which provides plugin interfaces for users and vendors to implement their own integrations with the key/certificate management solutions or signing service. Notation has the following plugins available for use now.
+
+- [AWS Signer plugin](https://docs.aws.amazon.com/signer/latest/developerguide/Welcome.html)
+- [Azure Key Vault plugin](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-sign-build-push)
+
+### Integration with admission controller for Kubernetes usage
+
+To enable users to verify and secure image deployment on Kubernetes, the Notary Project maintainers worked with the [Ratify](https://github.com/deislabs/ratify) and [Kyverno](https://kyverno.io/) team respectively to provide solutions for verifying images signed by Notation before deploying them to Kubernetes. Users will have two different options to build a complete end-to-end image integrity workflow for their environments. See the following two guides for details.
+
+- [Sign and verify an image with Notation, Ratify, and OPA Gatekeeper](https://ratify.dev/blog/sign-and-verify-image-with-notation-ratify)
+- [Verify CNCF Notary Project signatures with Kyverno](https://kyverno.io/docs/writing-policies/verify-images/notary/)
+
+![e2e workflow](/e2e-workflow.png)
+
+## Security audit
+
+The Notary Project also completed fuzzing audit and security audit in 2023. All vulnerabilities found during the audit are fixed in Notation v1.0.0 (and also present in the prior v1.0.0 RC-7 release). You can find two security audit reports as follows.
+
+- [Notation Security Audit Report 2023](https://github.com/notaryproject/notaryproject/blob/main/security/reports/audit/ADA-notation-security-audit-23.pdf)
+- [Notary Project fuzzing audit report 22-23](https://github.com/notaryproject/notaryproject/tree/main/security/reports/fuzzing/ADA-fuzzing-audit-22-23.pdf)
+
+## What's the next
+
+The following new features are planned for the upcoming milestones. Feel free to let us know of your ideas or use cases in the Slack channel or GitHub issues.
+
+- Sign and verify any file, even if distributed outside of an OCI registry
```

Comment:
I am not sure listing the features is the best approach. Especially in bulleted form as this makes it to concrete and sets the expectations that they will come in next (???) release. I would suggest using a paragraph to explain what the future plans are and properly set the expectations.

Reply:
Revised the introduction paragraph